observability/logging/logging_release_notes/cluster-logging-deploying.adoc

```diff
@@ -0,0 +1,48 @@
+:_mod-docs-content-type: ASSEMBLY
+:context: cluster-logging-deploying
+[id="cluster-logging-deploying"]
+= Installing Logging
+include::_attributes/common-attributes.adoc[]
+include::_attributes/attributes-openshift-dedicated.adoc[]
+
+toc::[]
+
+{Product-title} Operators use custom resources (CR) to manage applications and their components. High-level configuration and settings are provided by the user within a CR. The Operator translates high-level directives into low-level actions, based on best practices embedded within the Operator’s logic. A custom resource definition (CRD) defines a CR and lists all the configurations available to users of the Operator. Installing an Operator creates the CRDs, which are then used to generate CRs.
+
+[IMPORTANT]
+====
+You must install the {clo} *after* the log store Operator.
+====
+
+You deploy {logging} by installing the {loki-op} to manage your log store, followed by the {clo} to manage the components of logging. You can use either the {product-title} web console or the {product-title} CLI to install or configure {logging}.
+
+[TIP]
```

Comment:
The RH Supplementary guide advises that we avoid stacked notes. Could one of the notes be moved?

https://redhat-documentation.github.io/supplementary-style-guide/#admonitions

Reply:
As this is legacy content, i've added the suggestion to a follow-up Jira: https://issues.redhat.com/browse/OBSDOCS-1747. Note that in Logging 6.2, which is the current release the notes are not stacked: https://github.com/openshift/openshift-docs/blob/9bd9781fb1737e48ca3bdbb44a8edf771b0225fb/observability/logging/logging-6.2/6x-cluster-logging-deploying-6.2.adoc